xml/containers-get.xml

```diff
@@ -303,7 +303,108 @@
      </listitem>
     </varlistentry>
    </variablelist>
-  </sect2>
+  <sect3 xml:id="sec-bci-labels">
+    <title>&bci; labels</title>
+    <para>
+      &bci;s feature the following labels.
+    </para>
+    <variablelist>
+    <varlistentry>
+    <term>
+      com.suse.image-type
+    </term>
+    <listitem>
+    <para>
+      Indicates that the container image belongs to the &sle; product family.
```

Comment:
This one is not correct. It identifies whether this is a pure BCI or an application container based on another BCI